src/Hint/Extensions.hs

```diff
@@ -182,26 +182,37 @@ import Language.Haskell.GhclibParserEx.GHC.Hs.Pat
 import Language.Haskell.GhclibParserEx.GHC.Hs.Expr
 import Language.Haskell.GhclibParserEx.GHC.Hs.Types
 import Language.Haskell.GhclibParserEx.GHC.Hs.Decls
+import qualified Language.Haskell.GhclibParserEx.DynFlags as GhclibParserEx
+import GHC.LanguageExtensions.Type
+
+-- Terrible hack.
```

Comment:
Why does this hack exist? GHC must have mechanisms for parsing extension names, so why can't we reuse that?

Reply:
I'll look into it.

Reply:
Cracked it.